docs/README.md

```diff
@@ -1210,6 +1210,8 @@ Example **drop column** migrations:
 
 ### Drop constraint
 
+:warning: The **drop constraint** operation is deprecated. Please use the [drop multi-column constraint](#drop-multi-column-constraint) operation instead. The **drop_constraint** operation will be removed in a future release of `pgroll`. :warning:
```

Comment:
I wonder if we will rename drop multi column to this once deprecation is in?

Reply:
yes, I think so. But I don't think that needs to be part of this notice; it's a breaking change either way.

Reply:
unless you think we need to mention that `drop_multicolumn_constraint` will become the new `drop_constraint` here?